README.md

```diff
@@ -241,7 +241,7 @@ app.model({
   subscriptions: {
     emitWoofs: (send) => {
       // emit a woof every second
-      setInterval(() =>  send('print', { woof: 'meow?' }), 1000)
+      setInterval(() =>  send('print', { printWoofs: 'meow?' }), 1000)
```

Comment:
this won't fly because line `248` expects a `"woof"` property to exist on the object